libqtile/window.py

```diff
@@ -325,6 +325,8 @@ def update_state(self):
 
         state = self.window.get_net_wm_state()
 
+        hook.fire("window_state_changed", self)
```

Comment:
We're working on a very similar hook in parallel in #1798, but that has more arguments; can you add the state as an argument to this one if we end up adding it here?

Reply:
Sure, just added that.